biojava-structure/src/test/java/org/biojava/nbio/structure/secstruc/TestBasePairParameters.java

```diff
@@ -0,0 +1,31 @@
+import org.biojava.nbio.structure.secstruc.BasePairParameters;
+import org.junit.Test;
+
+/**
+ * Created by luke on 7/21/17.
+ */
+public class TestBasePairParameters {
```

Comment:
For the input of the method, I was wondering if it would be better to pass directly a `Nucleotide` Chain (or a `List` of `Nucleotide`), since the method only works with the nucleotide chains of the `Structure` object. It would also save you the `Nucleotide` chain extraction part of the code. This is only a suggestion, totally your choice!